draft-ietf-tls-tls13.md

```diff
@@ -4128,9 +4212,174 @@ Section 7 of [RFC6066], as it is not applicable to AEAD ciphers and has
 been shown to be insecure in some scenarios.
 
 
-#  Security Analysis
+#  Overview of Security Properties {#security-analysis}
+
+A complete security analysis of TLS is outside the scope of this document.
+In this section, we provide an informal description the desired properties
+as well as references to more detailed work in the research literature
+which provides more formal definitions.
+
+We cover properties of the handshake separately from those of the record layer.
+
+## Handshake {#security-handshake}
+
+The TLS handshake is an Authenticated Key Exchange (AKE) protocol which
+is intended to provide both one-way authenticated (server-only) and
+mutually authenticated (client and server) functionality. At the completion
+of the handshake, each side outputs its view on the following values:
+
+- A "session key" from which can be derived a set of working keys.
+- A set of cryptographic parameters (algorithms, etc.)
+- The identities of the communicating parties.
+
+We assume that the attacker has complete control of the network in
+between the parties {{RFC3552}}. Even under these conditions, the
+handshake should provide the properties listed below. Note that
+these properties are not necessarily independent, but reflect
+the protocol consumers' needs.
+
+Establishing the same session key.
+: The handshake needs to output the same session key on both sides of the
+handshake (See {{CK01}}; defn 1, part 1).
+
+Secrecy of the session key.
+: The shared session key should be known only to the communicating
+parties, not to the attacker (See {{CK01}}; defn 1, part 2).  Note that
+in a unilaterally authenticated connection, the attacker can establish
+its own session keys with the server, but those session keys are
+distinct from those established by the client.
+
+Peer Authentication.
+: The client's view of the peer identity should reflect the server's
+identity. If the client is authenticated, the server's view of the
+peer identity should match the client's identity.
+
+Uniqueness of the session key:
+: Any two distinct handshakes should produce distinct, unrelated session
+keys [REF: Need cite.]
+
+Downgrade protection.
+: The cryptographic parameters should be the same on both sides and
+should be the same as if the peers had been communicating in the
+absence of an attack (See {{BBFKZG16}}; defns 8 and 9}).
+
+Forward secret
```

Comment:
It seems like there's overlap between this and the uniqueness goal.  Either way, you're talking about information from one session not providing information about another session.  Perhaps there's a broader overall "independence of sessions" goal?, e.g.:

```
Independence of sessions.
: If the session key and all long-term authentication keys are compromised for a given session, this does not compromise the security of any other session. (This is a generalization of the common notion of Forward Secrecy, see {{DOW92}}.)
```